wit/postgres/README.md

```diff
@@ -0,0 +1,111 @@
+# 🐘 `wasmcloud:postgres` WIT interface
+
+This folder contains [WIT][wit] definitions for `wasmcloud:postgres`, an interface for interacting with the [Postgres database system][postgres] from WebAssembly.
+
+[wit]: https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md
+[postgres]: https://postgresql.org
+
+## 👟 Using this WIT interface
+
+While `wasmcloud:postgres` is best used from the [wasmCloud][wasmcloud] ecosystem (i.e. with the wasmCloud `sqldb-postgres` provider), this WIT contract can be used by any ecosystem project that supports WIT interfaces and fulfilling WIT contracts (whether locally, or in a distriuted fashion as wasmCloud does).
+
+[wasmcloud]: https://wasmcloud.com/docs/intro
+
+### 🏗️ Building WebAssembly components
+
+These definitions are meant to be used while *creating* WebAssembly components, with whatever language toolchain is available to you.
+
+| Language   | Toolchain                            |
+|------------|--------------------------------------|
+| Javascript | [`jco`][jco]  |
+| Rust       | [`cargo-component`][cargo-component] |
+| Rust       | [`wit-bindgen`][wit-bindgen-rust]    |
+| Python     | [`componentize-py`][compnentize-py]  |
+| Golang     | [`wit-bindgen`][wit-bindgen-go]      |
+
+Depending on which language and toolchain you use, the specifics differ, but you should end up with a project that contains a `wit` folder.
+
+[cargo-component]: https://github.com/bytecodealliance/cargo-component
+[compnentize-py]: https://github.com/bytecodealliance/componentize-py
+[jco]: https://github.com/bytecodealliance/jco
+[wit-bindgen-go]: https://github.com/bytecodealliance/wit-bindgen?tab=readme-ov-file#guest-tinygo
+[wit-bindgen-rust]: https://github.com/bytecodealliance/wit-bindgen
+
+### ⬇️ Downloading this WIT
+
+While ecosystem tooling for pulling and using WIT-manifest-only components develops, the easiest way to get started is to use [`wit-deps`][wit-deps], a dependency manager for WIT files.
+
+In your project, include the following `wit/deps.toml`:
+
+```yaml
+postgres = "https://github.com/vados-cosmonic/wit-wasmcloud-postgres/archive/v0.1.0-draft.tar.gz"
```

Comment:
Should this be the new location as a release asset?

Reply:
Ah, forgot to change this -- done!